docs/src/main/sphinx/installation/cli.rst

```diff
@@ -287,3 +293,124 @@ To view debug information, including the stack trace for failures, use the
+  * - :ref:`--keystore-path <cli-certificate-auth>`
+    - The location of the Java Keystore file that is used to secure TLS.
+  * - :ref:`--keystore-password <cli-certificate-auth>`
+    - The password for the keystore. This must match the password you specified
+      when creating the keystore.
+  * - ``--keystore-type``
+    - Keystore type
+  * - :ref:`--krb5-config-path <cli-kerberos-auth>`
+    - Kerberos configuration file.
+  * - :ref:`--krb5-credential-cache-path <cli-kerberos-auth>`
+    - Kerberos credential cache path
+  * - :ref:`--krb5-disable-remote-service-hostname-canonicalization <cli-kerberos-auth>`
+    - Disable service hostname canonicalization using the DNS reverse lookup
+  * - :ref:`--krb5-keytab-path <cli-kerberos-auth>`
+    - The location of the keytab that can be used to authenticate the principal
+      specified by ``--krb5-principal``
+  * - :ref:`--krb5-principal <cli-kerberos-auth>`
+    - The principal to use when authenticating to the coordinator.
+  * - :ref:`--krb5-remote-service-name <cli-kerberos-auth>`
+    - Trino coordinator Kerberos service name.
+  * - :ref:`--krb5-service-principal-pattern <cli-kerberos-auth>`
+    - Remote kerberos service principal pattern (default: ${SERVICE}@${HOST})
+  * - ``--network-logging``
+    - Network logging level [NONE, BASIC, HEADERS, BODY] (default: NONE)
+  * - :ref:`--output-format <cli-output-format>`
+    - Output format for batch mode [ALIGNED, VERTICAL, TSV, TSV_HEADER, CSV,
+      CSV_HEADER, CSV_UNQUOTED, CSV_HEADER_UNQUOTED, JSON, NULL] (default: CSV)
+  * - :ref:`--password <cli-username-password-auth>`
+    - Prompt for password
+  * - ``--progress``
+    - Show query progress in batch mode
+  * - ``--resource-estimate``
+    - Resource estimate (property can be used multiple times; format is
+      key=value)
+  * - ``--schema``
+    - Default schema
+  * - ``--server``
+    - The address and port of the Trino coordinator. The port must be set to
+      the port the Trino coordinator is listening for HTTPS connections on.
+      Trino server location (default: localhost:8080). The Trino CLI does not
+      support using ``http`` scheme for the URL when using LDAP authentication
+  * - ``--session``
+    - Session property (property can be used multiple times; format is
+      key=value; use 'SHOW SESSION' to see available properties)
+  * - ``--session-user``
+    - Username to impersonate
+  * - ``--socks-proxy``
+    - SOCKS proxy to use for server connections
+  * - ``--source``
+    - Name of source making query (default: trino-cli)
+  * - ``--timezone``
+    - Session time zone (default: America/New_York)
+  * - ``--trace-token``
+    - Trace token
+  * - ``--truststore-password``
+    - The password for the truststore. This must match the password you
+      specified when creating the truststore.
+  * - ``--truststore-path``
+    - The location of the Java truststore file that will be used to secure TLS.
+  * - ``--truststore-type``
```

Comment:
Should the `--truststore` related options be mentioned in the TLS/HTTPS section at the top of the doc?